domino/api/src/main/java/io/quarkus/domino/ProjectDependencyResolver.java

```diff
@@ -1247,7 +1258,7 @@ private void processNodes(DependencyNode node, int level, boolean remaining) {
         }
 
         final ArtifactCoords coords = toCoords(node.getArtifact());
-        if (isExcluded(coords)) {
+        if (!isIncluded(coords)) {
```

Comment:
this is the actual logic change, checking if the coords are in the inclusion list or not in the exclusion list.